javascript/javascript-import-and-require.md

```diff
@@ -0,0 +1,79 @@
+# JavaScript - Require & Import
+
+## Project Time
+
+About 45 Minutes
+
+- 30 minutes for the video and reference material
+- 10 minutes for Check for Understanding
+- 5 minutes for verify
+
+## Prerequisites
+
+- JavaScript 7
+- [JavaScript Import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import)
+- [JavaScript Require](https://nodejs.org/en/knowledge/getting-started/what-is-require/)
+
+## Motivation
+
+JavaScript is one of most used language. It is lightweight, Interpreted and Just-in-time compiled programming language with first-class function. While it is most well-known as the scripting language for webpage, Many no-browser environmemts also use it. JavaScript is a prototype-based, multi-paradigm, single-threaded, dynamic language, supporting object-oriented, imperative and declarative style.
+
+## Objectives
+
+- Create understanding in user regarding of import and require
+
+## Supplemental Materials

```

Comment:
Please avoid using any reference or links as this is a mini-lesson.

Reply:
what do you mean @Aayush-1999 ?

Reply:
@alodahl I am suggesting to remove supplement materials.

Reply:
Ok, I think I see what is happening.  Either these are:
1. intended to be required links for apprentices.  If this is the case, this section should be called ## Materials, because they are not supplemental. Also, tell them how long to spend looking at each link, and what to look at or notice if necessary.
2.  not required, but indeed supplemental.  If this is the case, these need to be moved to the bottom of the outline as the last section.